diff --git a/src/pmcrypto/openpgp.ts b/src/pmcrypto/openpgp.ts
index 923c953e9ba2881fd52e976a593bf66208159565..d2b4360fdfeb3728ff77a6d66b1103b6315683fe 100644
--- a/src/pmcrypto/openpgp.ts
+++ b/src/pmcrypto/openpgp.ts
@@ -1,4 +1,4 @@
-import { config } from "openpgp/lightweight";
+import { config } from "openpgp";
 
 export const setConfig = () => {
     config.s2kIterationCountByte = 255;
@@ -34,4 +34,4 @@ export const setConfig = () => {
     config.maxArgon2MemoryExponent = 20;
 };
 
-export * from "openpgp/lightweight";
+export * from "openpgp";
diff --git a/src/pmcrypto/pmcrypto.d.ts b/src/pmcrypto/pmcrypto.d.ts
index 82052deea448f3d6f17561335fcc2a2a0ffc6062..1da4e0bcc457450727af920106f5b94440fd90cc 100644
--- a/src/pmcrypto/pmcrypto.d.ts
+++ b/src/pmcrypto/pmcrypto.d.ts
@@ -23,7 +23,7 @@ import {
     readPrivateKey,
     type PartialConfig,
     type AlgorithmInfo,
-} from "openpgp/lightweight";
+} from "openpgp";
 
 import { VERIFICATION_STATUS, SIGNATURE_TYPES } from "./constants.ts";
 import type {
